### PR TITLE
chaos monkey migration

### PR DESCRIPTION
**Pull request by zakqavalley**

### CODE REVIEW

**Review by enyambudzi — APPROVED** [review]
LGTM

**Review by ranbeersingh1 — APPROVED** [review]
LGTM

File: testing/chaosmonkey/batch.tf [hunk 1]
```diff
@@ -40,8 +40,11 @@ resource "aws_batch_compute_environment" "batch_ce" {
 
     type = "EC2"
 
-    tags = var.tags
-  }
+    tags = merge(
+    var.tags,
+    {
+      "Name" = "${local.name_prefix}-asg"
+    },
```
Comment (bensonarokiadoss) [inline testing/chaosmonkey/batch.tf:47]: @zakqavalley Can you remove the extra , and close the merge function

Comment (smjmoj) [reply testing/chaosmonkey/batch.tf:47]: AS Benson said.
Also should that suffix be "-batch" rather than "-asg"?

**Pull request merged**